pkgs/development/python-modules/flask-security-too/default.nix

```diff
@@ -92,12 +94,12 @@ buildPythonPackage rec {
     mfa = [
       cryptography
       phonenumbers
+      webauthn
```

Comment:
qrcode was moved from common to mfa.

https://github.com/Flask-Middleware/flask-security/blob/5.3.0/pyproject.toml#L57